src/System/Relay/createRelaySSREnvironment.ts

```diff
@@ -125,20 +133,6 @@ export function createRelaySSREnvironment(config: Config = {}) {
     loggingEnabled && loggerMiddleware(),
     loggingEnabled && metaphysicsExtensionsLoggerMiddleware(),
     loggingEnabled && errorMiddleware({ disableServerMiddlewareTip: true }),
-
-    ...(getENV("ENABLE_QUERY_BATCHING")
-      ? [
-          batchMiddleware({
-            headers: authenticatedHeaders,
```

Comment:
This needs to set the headers properly (same as above) - decided to just rip this out as it is not enabled.